package.json

```diff
@@ -39,26 +41,20 @@
   "devDependencies": {
     "@most/eslint-config-most": "^1.0.2",
     "assert": "^1.3.0",
-    "babel-cli": "^6.5.1",
-    "babel-core": "^6.4.5",
-    "babel-eslint": "^6.0.0",
-    "babel-plugin-transform-es2015-modules-umd": "^6.4.3",
-    "babel-preset-es2015": "^6.5.0",
-    "babel-register": "^6.5.2",
+    "buba": "^2.0.1",
     "eslint": "^2.2.0",
     "eslint-config-standard": "^5.1.0",
     "eslint-plugin-promise": "^1.1.0",
     "eslint-plugin-standard": "^1.3.2",
-    "isparta": "^4.0.0",
+    "istanbul": "^1.1.0-alpha.1",
     "jsinspect": "^0.7.2",
     "mocha": "^2.4.5",
-    "most": "^0.19.0",
-    "rollup": "^0.25.2",
+    "rollup-plugin-buble": "^0.12.1",
```

Comment:
i think you are missing `rollup` itself for the `build-dist` script seems thats why travis-ci is failing